pkg/cli/clipboard.go

```
@@ -5,6 +5,16 @@ import (
 	"strings"
 )
 
+var clipboardReadFunc = readFromClipboard
+
+func readFromClipboard() (string, error) {
+	output, err := exec.Command("pbpaste").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)